### PR TITLE
Add hint to release notes on gh-pages to README.md

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:
We now add an additional badge to the README as well as a custom note
about the release notes.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:

None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +28,9 @@ Key:
 * `✓` Changes in main Kubernetes repo about CRI are actively implemented in CRI-O
 * `=` Maintenance is manual, only bugs will be patched.
 
+The release notes of CRI-O are hand-crafted and can be continuously retrieved
```
Comment (TomSweeneyRedHat) [inline README.md]: nit "of CRI-O" -> "for CRI-O"

Comment (saschagrunert) [reply README.md]: Fixed :+1:

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Comment by rhatdan** [comment]
Does this actually need to pass tests? Or just make sure the linting passes?

**Comment by saschagrunert** [comment]
> Does this actually need to pass tests? Or just make sure the linting passes?

Since it's more or less just documentation and the linter fix I'd say we can merge it. (Beware, I'm the author :smiling_imp:)

**Pull request merged**